.github/workflows/lint-codeql.yaml

```diff
@@ -15,7 +15,13 @@ on:
 permissions: read-all
 
 jobs:
+  check_repo:
+    name: Only run on the cilium/cilium repo
+    if: ${{ github.repository_owner == 'cilium' && github.repository_name == 'cilium' }}
+    runs-on: ubuntu-18.04
+
   check_changes:
+    needs: check_repo
```

Comment:
That won't be enough. It defines a dependency between the jobs but doesn't use the dependency in `check_changes`.